### ISSUE TITLE
chromedriver_2.33.exe hangs around after test finishes. 

### ISSUE THREAD

**Issue by dlfu** [angular/protractor#4572]
Bug report
---

when running `webdriver-manager update` and `protractor config.js` the test processes properly, but upon finishing, leaves behind a chrome_2.33.exe and never cleans it up.

- Node Version: `8.7.0`
- Protractor Version: `5.2.0`
- Angular Version: `1.5.8`
- Browser(s): `chrome 62`
- Operating System and Version `Windows 10`
- protractor configuration file: very simple, no other settings other than listed

  ```
  framework: "jasmine",
  plugins: [],
  jasmineNodeOpts: {
	  defaultTimeoutInterval: 120000
  },
  capabilities: { 
	  "browserName": "chrome", 
	  "chromeOptions": {
	  	  "args": ["--window-size=1200,1000"]
	  }
  },

  suites: {
	  example: "spec.js"
  }

this causes a new chromedriver.exe to pile up after each time we test. and makes it impossible for us to do a clean build without having to kill all the processes beforehand

**Comment by acuper** [comment]
I had the same issue, downgrade of server helped:
webdriver-manager update --versions.standalone 3.4.0

**Comment by dlfu** [comment]
yep that seems to have fixed it. @cnishina you mentioned a fix in a [stackoverflow answer](https://stackoverflow.com/questions/46965756/protractor-leaves-behind-a-chromedriver-exe?noredirect=1#comment80971264_46965756) This seems to have been broken from 3.4.0 to latest (3.7.0)

**Comment by ezekeal** [comment]
This issue can be reporoduced with the config file and spec files here on Windows 10
it uses both
**directConnect**
and
**restartBrowserBetweenTests**
[spec and config on pastebin](https://pastebin.com/eAGtTtp5)

**Comment by wswebcreation** [comment]
Is this issue still valid with the latest version of chromedriver?

**Comment by vsravuri** [comment]
I see this issue with Chromedriver 2.34

**Comment by ezekeal** [comment]
Our issue was resolved by removing the flag `--no-sandbox` from our test config.

**Comment by sisbilir** [comment]
I still see the issue with Chromedriver 2.35

**Comment by johnflux** [comment]
I can confirm it still happens in chromedriver 2.35  and selenium 3.8.1

**Comment by dlfu** [comment]
Still an issue in 2.39
